.gitignore

```diff
@@ -180,3 +180,5 @@ yarn-debug.log*
 yarn-error.log*
 .npm/
 coverage_js/
+
+*.png
```

Comment:
Not sure about this change -- I bet there are already some PNGs in the repository (docs, logos, ...).

Reply:
i'll remove that from .gitignore.